src/setup/platformSetup/index.website.js

```diff
@@ -1,4 +1,8 @@
 import {AppRegistry} from 'react-native';
+
+// This is a polyfill for InternetExplorer to support the modern KeyboardEvent.key and KeyboardEvent.code instead of KeyboardEvent.keyCode
+import 'shim-keyboard-event-key';
```

Comment:
I don't feel like that IE is worth supporting.

Reply:
Haha fair enough, but this library is 2KB and super stable so I figured there's no harm in including it.